strings/phobia.json

```diff
@@ -298,5 +298,16 @@
 		"glockroach",
 		"bee",
 		"sting"
+	],
+"ocky icky": [
+		"admin",
+		"admins",
+		"antagonist",
+		"antag",
+		"ban",
+		"banned",
+		"reddit",
+		"round",
```

Comment:
man that circle sure is _starts shaking and crying_

Reply:
My belly is circular and full